src/fontra/views/editor/panel-selection-info.js

```diff
@@ -194,10 +194,15 @@ export default class SelectionInfoPanel extends Panel {
           fieldX: {
             key: '["leftMargin"]',
             value: glyphController.leftMargin,
+            disabled: glyphController.leftMargin == undefined,
             getValue: (layerGlyph, layerGlyphController, fieldItem) => {
               return layerGlyphController.leftMargin;
             },
             setValue: (layerGlyph, layerGlyphController, fieldItem, value) => {
+              if (layerGlyphController.leftMargin === undefined) {
```

Comment:
Btw. in which case can this be called? Isn't the input disabled in this case?

Reply:
Good point. I had this in place, before I added the 'disabled' functionality. Should I remove it?

Reply:
Yes please, this code is effectively unreachable, therefore better not there to begin with.